app-registry.yaml

```diff
@@ -1394,99 +1394,204 @@ apps:
-            \ cert-manager.io/cluster-issuer: letsencrypt-prod\n         nginx.ingress.kubernetes.io/secure-backends:\
-            \ \"true\"\n      hosts:\n         - ~\n{% if not (rancher_server | ipaddr)\
-            \ %}\n         - \"{{ rancher_server }}\"\n      tls:\n         - secretName:\
-            \ \"{{ rancher_server | replace('.', '-') }}-key\"\n           hosts:\n\
-            \             - \"{{ rancher_server }}\"\n{% endif %}\nprometheus:\n   persistence:\n\
-            \      storageClass: \"nfs\"\n   grafana:\n      adminPassword: {{ random_client_secret\
-            \ }}\n      grafana.ini:\n         server:\n            root_url: \"https://{{\
-            \ rancher_server }}/grafana\"\n         auth.generic_oauth:\n          \
-            \  auth_url: \"https://{{ rancher_server }}/grafana/auth/realms/master/protocol/openid-connect/auth\"\
-            \n            token_url: \"https://{{ rancher_server }}/grafana/auth/realms/master/protocol/openid-connect/token\"\
-            \n            api_url: \"https://{{ rancher_server }}/grafana/auth/realms/master/protocol/openid-connect/userinfo\"\
-            \n      ingress:\n         annotations:\n            kubernetes.io/tls-acme:\
-            \ \"true\"\n            cert-manager.io/cluster-issuer: letsencrypt-prod\n\
-            \            nginx.ingress.kubernetes.io/secure-backends: \"true\"\n   \
-            \      hosts:\n            - ~\n{% if not (rancher_server | ipaddr) %}\n\
-            \            - \"{{ rancher_server }}\"\n         tls:\n            - secretName:\
-            \ \"{{ rancher_server | replace('.', '-') }}-key\"\n              hosts:\n\
-            \                - \"{{ rancher_server }}\"\n{% endif %}\nkeycloak:\n  \
-            \ keycloak:\n       persistence:  \n         deployPostgres: false\n   \
-            \      dbVendor: postgres\n         dbName: keycloak\n         dbPort: 5432\n\
-            \         dbHost: cloudman-postgresql\n         dbUser: admin\n        \
-            \ existingSecret: cloudman-postgres-superuser-password\n         existingSecretKey:\
-            \ postgres-superuser-password\n       password: \"{{ rancher_password }}\"\
-            \n       ingress:\n         enabled: true\n         path: /auth\n      \
-            \   annotations:\n            kubernetes.io/tls-acme: \"true\"\n       \
-            \     cert-manager.io/cluster-issuer: letsencrypt-prod\n            nginx.ingress.kubernetes.io/secure-backends:\
-            \ \"true\"\n         hosts:\n            - ~\n{% if not (rancher_server\
-            \ | ipaddr) %}\n            - \"{{ rancher_server }}\"\n         tls:\n\
-            \            - secretName: \"{{ rancher_server | replace('.', '-') }}-key\"\
-            \n              hosts:\n                - \"{{ rancher_server }}\"\n{% endif\
-            \ %}\nglobal:\n   domain: \"{{ rancher_server }}\""
+          cm_boot_image: almahmoud/cloudman-boot:latest
```

Comment:
This should also be updated right? There were a couple of other places too.

Reply:
No? This is for my dev branch, I'm keeping `am-dev` pointing to my forks/images

Reply:
Oh sorry, didn't notice that.